teachertool/src/transforms/confirmAsync.ts

```diff
@@ -0,0 +1,7 @@
+export async function confirmAsync(prompt: string) {
```

Comment:
We had a discussion about helpers versus transforms in Thomas's PR and I feel like this is a similar stream.. This feels more like a helper to me than a state manager. Did we want to make a helpers folder or put something like this in utils? Or am I misunderstanding and this does manipulate state?

Reply:
This is clearly not a helper in my view. A helper makes reading/writing data easier. I put it this transforms because it shows UI. I don't want to get to fine-grained or precise with these categories.